simtools/model/camera.py

```diff
@@ -164,25 +166,19 @@ def _rotate_pixels(self, pixels):
         # (when looking from the camera onto the dish),
         # and the z-axis points in any case from (primary) dish towards camera.
         # To get the camera for an observer facing the camera, need to rotate by 90 degrees.
-        rotate_angle += np.deg2rad(90)
+        rotate_angle += (90 * u.deg).to("rad")
```

Comment:
Yes, it works because "rad" and u.rad are equivalent. However, since I have to make a change to the log, I will change this here as well in order to save the translation.